SetupConan.cmake

```diff
@@ -17,16 +17,26 @@ if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
 elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
     message(STATUS "Setup dependencies with conan (gcc ${COMPILER_VERSION_MAJOR}.${COMPILER_VERSION_MINOR})")
     set(CONAN_COMPILER "gcc")
+elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "AppleClang")
+    message(STATUS "Setup dependencies with conan (apple clang ${COMPILER_VERSION_MAJOR}.${COMPILER_VERSION_MINOR})")
 else()
     # break cmake execution
     set(CONAN_INSTALL_CMD "conan install .. <list-of-settings>")
     message(FATAL_ERROR "Dependencies are not configured. Install dependencies first, e.g. running in build dir: ${CONAN_INSTALL_CMD}")
 endif()
+
 message(STATUS "Using the following toolchain for conan: CXX=${CMAKE_CXX_COMPILER}, CC=${CMAKE_C_COMPILER}")
-set(CONAN_INSTALL_CMD conan install ${CMAKE_CURRENT_SOURCE_DIR} -if ${CMAKE_BINARY_DIR} -s arch=x86_64 -s os=Linux -s compiler=${CONAN_COMPILER} -s compiler.version=${COMPILER_VERSION_MAJOR}.${COMPILER_VERSION_MINOR} -s compiler.libcxx=libstdc++11 -s build_type=Release -e CXX=${CMAKE_CXX_COMPILER} -e CC=${CMAKE_C_COMPILER} --build missing)
+
+if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "AppleClang")
```

Comment:
@Morheit, you don't need separate if/else. Set unstall cmd in the if/else above

Reply:
@malirod, done